presto-main/src/main/java/io/prestosql/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -373,7 +374,7 @@ protected Node visitShowCreate(ShowCreate node, Void context)
                 }
 
                 Query query = parseView(viewDefinition.get().getOriginalSql(), objectName, node);
-                String sql = formatSql(new CreateView(createQualifiedName(objectName), query, false, Optional.empty()), Optional.of(parameters)).trim();
+                String sql = formatSql(new CreateView(createQualifiedName(objectName, MetadataUtil.getDelimitedList(node.getName())), query, false, Optional.empty()), Optional.of(parameters)).trim();
```

Comment:
This seems unnecessary. It looks like you could just do: `new CreateView(node.getName(), ...)` instead of trying to recreate the `QualifiedName` from the `QualifiedObjectName` that was derived from `node.getName()`.

If you do that, you should also be able to remove `MetadataUtil.getDelimitedList()`, which I'm not fond of.